lib/inject.rb

```diff
@@ -0,0 +1,42 @@
+class Array
+  def my_inject(symbol,*start)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for my_inject is too high. [16.91/15]